### PR TITLE
feat: support esm in browser

### PR DESCRIPTION
**Pull request by BuptStEve**
**Summary**

v0.2.1-0

Demo: https://codepen.io/buptsteve/pen/xNBJzy

* chore: only publish src/ and dist/, .babelrc -> babel.config.js
* chore: support esm in browser

```html
<!-- unpkg -->
<script type="module">
    import { lock, unlock } from 'https://unpkg.com/tua-body-scroll-lock@0.2.1-0/dist/tua-bsl.esm.browser.min.js'

    lock()
    unlock()
</script>

<!-- jsdelivr -->
<script type="module">
    import { lock, unlock } from 'https://cdn.jsdelivr.net/npm/tua-body-scroll-lock@0.2.1-0/dist/tua-bsl.esm.browser.min.js'

    lock()
    unlock()
</script>
```

**What kind of change does this PR introduce?** (check at least one)

- [x] Bug fix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Docs
- [x] Build-related changes
- [ ] Other, please describe:

**The PR fulfills these requirements:**

- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following platforms: (Providing a detailed version will be better.)

- [x] PC
- [x] iOS
- [x] Android

### CODE REVIEW

**Comment by BuptStEve** [comment]
cc @Grawl

**Comment by Grawl** [comment]
never tried ESM in browser

**Comment by Grawl** [comment]
works on iPhone.

File: README-zh_CN.md [hunk 1]
```diff
@@ -34,8 +34,11 @@ $ yarn add tua-body-scroll-lock
 
 ### CDN
 * UMD(`tua-bsl.umd.js`)
-  * [unpkg](https://unpkg.com/tua-body-scroll-lock)
-  * [jsdelivr](https://cdn.jsdelivr.net/npm/tua-body-scroll-lock)
+  * unpkg: https://unpkg.com/tua-body-scroll-lock
+  * jsdelivr: https://cdn.jsdelivr.net/npm/tua-body-scroll-lock
```
Comment (Grawl) [inline README-zh_CN.md]: `+` for this

Comment (BuptStEve) [reply README-zh_CN.md]: what?

Comment (Grawl) [reply README-zh_CN.md]: i appreciate this change, just want to "like it"

**Pull request merged**